OpenSim/Actuators/Millard2012AccelerationMuscle.h

```diff
@@ -51,7 +51,39 @@ The parent class, Muscle.h, provides
 
 namespace OpenSim {
 
+//==============================================================================
+//                   MILLARD2012ACCELERATIONMUSCLE EXCEPTIONS
+//==============================================================================
+class Millard2012AccelerationMuscleCannotEquilibrate : public Exception {
```

Comment:
Why is this Exception specific to Millard2012AccelerationMuscle? There could be a single `MuscleCannotEquilibrate` or `MuscleCannotInitialize` Exception defined by Muscle that all muscles could use.